editor/editor_help.cpp

```diff
@@ -215,7 +215,7 @@ void EditorHelp::_search(bool p_search_previous) {
 
 void EditorHelp::_class_desc_finished() {
 	if (scroll_to >= 0) {
-		class_desc->scroll_to_paragraph(scroll_to);
+		class_desc->connect("draw", callable_mp(class_desc, &RichTextLabel::scroll_to_paragraph).bind(scroll_to), CONNECT_ONE_SHOT | CONNECT_DEFERRED);
```

Comment:
```suggestion
		class_desc->connect(SceneStringName(draw), callable_mp(class_desc, &RichTextLabel::scroll_to_paragraph).bind(scroll_to), CONNECT_ONE_SHOT | CONNECT_DEFERRED);
```

Reply:
I wonder if a `call_deferred()` is enough now, without connect to the `draw` signal.

Reply:
Thread can't directly update scrollbars, so it is done in the draw. This change is correct (scrolling after `finished` and first redraw), but we might want to add another signal instead of directly connecting to the `draw` (`finished` probably should not be changed, since other code might expect it to fire even for hidden RTL that will never redraw).

Reply:
@ydeltastar WDYT about the above suggestion? Do you want to implement that?

Reply:
Sounds good. I added a signal that triggers after loading, processing, and validation for drawing. It also needed an equivalent method for 'is_finished()' so the `EditorHelp` could use the signal correctly.

Reply:
I'm very perplexed on the addition of another signal. It sounds "needlessly" confusing. I'd be wary as developers may use one or the other without understanding the difference.

Reply:
Yeah, all it does is simulate a one-shot connection to `draw`. It is very specific, can only happen in the first draw after `finished` so it could be just a note in the documentation as well.

Reply:
Maybe you can connect to `finished` and then in the callback connect to `draw`?

Reply:
It's what I did in the first iteration. I reverted the PR to it and added a note to the `finished` signal.

Reply:
Yeah, simpler solutions are better. No point in adding a signal if it does the same.